### PR TITLE
EJ2-Documentation-850455-Ug-Documentation-For-Batch-Editing

### PR DESCRIPTION
**Pull request by harinikrh**

### CODE REVIEW

File: ej2-javascript/grid/editing/batch-editing.md [hunk 1]
```diff
@@ -40,13 +46,11 @@ In Batch edit mode, when you double-click on the grid cell, then the target cell
 {% previewsample "page.domainurl/code-snippet/grid/grid-cs17" %}
 {% endif %}
 
-> If a column's [allowEditing](../../api/grid/column/#allowediting) property is set to false, then the focus can be skipped in that non-editable column by clicking the tab or shift-tab key while in batch edit mode.
-
 ## Automatically update the column based on another column edited value in batch mode
```
Comment (VigneshNatarajan27) [inline ej2-javascript/grid/editing/batch-editing.md:49]: "in batch mode" is not necessary

File: ej2-javascript/grid/editing/batch-editing.md [hunk 2]
```diff
@@ -75,11 +79,19 @@ In the below demo, we have update the `TotalCost` column value based on the `Uni
 {% previewsample "page.domainurl/code-snippet/grid/grid-cs18" %}
 {% endif %}
 
-## Cancel edit based on condition in batch mode
+> * You can utilize the [updateCell](../../api/grid/#updatecell) property to update cells in batch mode.
```
Comment (VigneshNatarajan27) [inline ej2-javascript/grid/editing/batch-editing.md:82]: not properly, updateCell is method

File: ej2-javascript/grid/editing/batch-editing.md (hunk 1 above)
Comment (harinikrh) [reply ej2-javascript/grid/editing/batch-editing.md]: removed and modified the content

File: ej2-javascript/grid/editing/batch-editing.md (hunk 2 above)
Comment (harinikrh) [reply ej2-javascript/grid/editing/batch-editing.md]: changed  property in to method

**Review by VigneshNatarajan27 — APPROVED** [review]
changes are fine

**Pull request merged**